back-end/routers/productsRoute.js

```diff
@@ -0,0 +1,16 @@
+const express = require("express");
+const router = express.Router();
+
+const { authenticate } = require("../services/productsService");
+const { generalResponse } = require("../helpers/responses");
+const { mockData, mockError } = require("../helpers/mock-data");
+
+router.route("/").get((req, res) => {
+  const { headers } = req;
+  if (authenticate(headers)) {
+    return generalResponse(200, res, mockData);
```

Comment:
@Jeffzholy generally the better practice is to switch the 2 things: if not authenticated then do something, but the main part of the handler should be the normal valid response

Reply:
`generalResponse` is not going to return anything and right now it's not worth to hide the functionality behind a method when you can just have here: 
```
res.status(200).send(mock.data);
return;
```